src/file.cpp

```diff
@@ -60,6 +62,8 @@ POSSIBILITY OF SUCH DAMAGE.
 #pragma GCC diagnostic pop
```

Comment:
I didn't want to add `-Wunused-macros` to the `disable_warnings_push.hpp`, not sure if it's a good idea